### ISSUE TITLE
First tab is too much near the window border

### ISSUE THREAD

**Issue by maicol07** [black7375/Firefox-UI-Fix#229]
**Describe the bug**
See title and screenshot

**Expected behavior**
First tab has some margin from the left border of the window

**Screenshots**
![immagine](https://user-images.githubusercontent.com/9463142/134925953-d82778d7-4393-4603-a8fe-42c4b633410d.png)

**Environment:**

 - Distribution
   - [ ] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
   - [x] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)
 - Firefox Version: 
 - OS:
   - [ ] Linux
   - [x] Windows
   - [ ] Mac
   - [ ] Other: 
 - Theme:
   - [x] Light
   - [ ] Dark
   - [ ] Alpenglow (System: Light)
   - [ ] Alpenglow (System: Dark)
   - [ ] System Default
     - GTK Theme Name (Linux only): 
   - [ ] Other: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

**Additional context**

**Comment by BPower0036** [comment]
Maybe you can use this code
```
  /** Gap before First Tab **********************************************************************************/
  :root:is([sizemode="maximized"], [sizemode="fullscreen"]) #tabbrowser-tabs:not([positionpinnedtabs]) {
    padding-left: 7px !important;
  }
```

**Comment by black7375** [comment]
Will be adjustable with variables in the next version.

https://github.com/black7375/Firefox-UI-Fix/blob/dabc91cde340d5a85d092ba03bcdce12db9e6f4e/userChrome.css#L1120-L1123

**Default**
![image](https://user-images.githubusercontent.com/25581533/136699910-8350f814-7eed-457f-a50a-9fa96432dfb6.png)

**Set to `8px`**
![image](https://user-images.githubusercontent.com/25581533/136699866-94591ba3-7ee6-4aa5-956f-e25dea40e2a9.png)

Thanks to @BPower0036 for replying in advance and providing the code.
